From 11704324287f9f552e1524f4a680b8744abd0361 Mon Sep 17 00:00:00 2001
From: Kai Kang <kai.kang@windriver.com>
Date: Fri, 9 Jun 2017 11:04:56 +0800
Subject: [PATCH] crmsh: disable network access for xsltproc when building

Upstream-Status: Inappropriate [embedded specific]

Pass option '--catalogs' to xsltproc via a2x to avoid network access.
And disble xmllint check by option '-L'.

Add option '--install-lib' of python setup.py to install python libraries to
right place.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

---
 Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 1ffe9c3..3d045e7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -45,7 +45,7 @@ endif
 	$(ASCIIDOC) --unsafe --backend=xhtml11 $<

 %.8: %.8.adoc
-	a2x -f manpage $<
+	a2x --xsltproc-opts=--catalogs --xsltproc-opts=--nonet --xsltproc-opts=--novalid -L -f manpage $<

 # Shared data files
 install-data-hook:
